Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -125,25 +137,29 @@ def map_orca_score_to_demisto_score(orca_score: int) -> Union[int, float]:  # py
 def get_incident_from_alert(alert: Dict[str, Any]) -> Dict[str, Any]:
     if alert is None:
         return {}
+
+    last_seen_time = dateutil.parser.parse(alert.get('state', {}).get('last_seen')).isoformat()
     return {
         'name': alert.get('state', {}).get('alert_id'),
-        'occurred': datetime_to_string(
-            datetime.strptime(alert.get('state', {}).get('last_seen'), "%Y-%m-%dT%H:%M:%S%z").isoformat()),
+        'occurred': last_seen_time,
         'rawJSON': json.dumps(alert),
         'severity': map_orca_score_to_demisto_score(orca_score=alert.get('state', {}).get('score'))
     }
 
 
 def get_incidents_from_alerts(alerts: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+    demisto.info("get_incidents_from_alerts enter")
     incidents = []
     for alert in alerts:
         alert['demisto_score'] = map_orca_score_to_demisto_score(orca_score=alert.get("state", {}).get("score", 1))
         incident = get_incident_from_alert(alert=alert)
         incidents.append(incident)
+
+    demisto.info("get_incidents_from_alerts done")
```

Comment:
Same question here.

Reply:
I want to keep this in case I get an error in the function